transaction/core/src/memo.rs

```diff
@@ -178,7 +181,7 @@ impl MemoPayload {
         let (key, nonce) = Split::<u8, U32>::split(okm);
 
         // Apply AES-256 in counter mode to the buffer
-        let mut aes256ctr = Aes256Ctr::from_block_cipher(Aes256::new(&key), &nonce);
+        let mut aes256ctr = Aes256Ctr::new(&key, &nonce);
```

Comment:
While this changed, it matches the usage at https://github.com/mobilecoinfoundation/mobilecoin/blob/f142750ae329fd94bb0cbc480d4d8019bf182763/fog/ocall_oram_storage/trusted/src/lib.rs#L330 (the master branch at the time of this writing)